cmd/oras/cp.go

```diff
@@ -139,7 +139,7 @@ func runCopy(opts copyOptions) error {
 	if dstRef := opts.To.Reference; dstRef == "" {
 		desc, err = oras.Resolve(ctx, src, opts.From.Reference, rOpts)
 		if err != nil {
-			return err
+			return fmt.Errorf("failed to resolve %s: %w", src, err)
```

Comment:
```suggestion
			return fmt.Errorf("failed to resolve %s: %w", opts.From.Reference, err)
```
A new ` oras.Resolve` call is added in line 154, better to copy the solution there also.